gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/users/OrganizationUsersResource.java

```diff
@@ -163,7 +163,7 @@ public void create(
     @ApiResponse(responseCode = "500", description = "Internal server error")
     public void handleBulkOperation(
             @PathParam("organizationId") String organizationId,
-            @Valid @NotNull @Schema(name = "bulkUserRequest", oneOf = {BulkCreateOrganizationUser.class, BulkDeleteOrganizationUser.class}) final BulkRequest.Generic bulkRequest,
+            @Valid @NotNull @Schema(name = "bulkUserRequest", oneOf = {BulkCreateOrganizationUser.class, BulkUpdateUser.class, BulkDeleteOrganizationUser.class}) final BulkRequest.Generic bulkRequest,
```

Comment:
Shouldn't it be BulUpdateOrganizaionUser?

Reply:
Ok I see.
WDYT about moving all specific model classes to single place? like as a inner classes in BulkRequest or along to it in **bulk** package?